user/user.js

```diff
@@ -0,0 +1,40 @@
+const constants = require('../utils/constants')
+
+const getUser = (req, res) => {
+  const token = req.body.token;
+  const responseData = getUserData(token);
+  res.status(responseData.status).send(responseData.data);
+}
+
+
+function getUserData(token) {
+
+  const unauthorized = 401;
+  const ok = 200;
+  const response = {
+    status: ok,
```

Comment:
It would be great to use some package for Http status processing. 
For example : [http-status-codes](https://www.npmjs.com/package/http-status-codes)

Reply:
done